src/main/java/com/dain_review/domain/post/service/FollowService.java

```diff
@@ -0,0 +1,234 @@
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.dain_review.global.util.ImageFileValidUtil.isValidImageFile;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class FollowService {
+
+    private final PostRepository postRepository;
+    private final UserRepository userRepository;
+    private final AttachedFileRepository attachedFileRepository;
+
+    private final ApplicationEventPublisher eventPublisher;
+    private final S3Util s3Util;
+
+    private final String S3_PATH_PREFIX = S3PathPrefixType.S3_FOLLOW_PATH.toString();
+
+    public FollowResponse createPost(
+            Long userId, FollowRequest followRequest, List<MultipartFile> imageFiles
+    ) {
+        User user = getUser(userId);
+
+        Post post =
+                Post.builder()
+                        .user(user)
+                        .categoryType(followRequest.categoryType())
+                        .title(followRequest.title())
+                        .content(followRequest.content())
+                        .followType(followRequest.followType())
+                        .build();
+
+        PostMeta postMeta =
+                PostMeta.builder()
+                        .post(post)
+                        .viewCount(0L) // 조회수와 댓글수는 0으로 초기화
+                        .commentCount(0L)
+                        .build();
+
+        post.setPostMeta(postMeta);
+        postRepository.save(post);
+
+        // 이미지 저장 및 url 반환
+        saveImageFiles(imageFiles, post);
+        List<String> imageUrls = findImageUrls(post.getId());
+
+        return FollowResponse.responseWithoutContentPreview(post, imageUrls);
+    }
+
+    public FollowResponse updatePost(
+            Long userId, Long postId, FollowRequest followRequest, List<MultipartFile> imageFiles
+    ) {
+        User user = getUser(userId);
+
+        Post existingPost =
+                postRepository
+                        .findById(postId)
+                        .orElseThrow(() -> new PostException(PostErrorCode.POST_NOT_FOUND));
+
+        if (!existingPost.getUser().getId().equals(user.getId())) {
+            throw new PostException(PostErrorCode.UNAUTHORIZED_ACCESS);
+        }
+
+        PostMeta existingPostMeta = existingPost.getPostMeta();
+
+        Post updatedPost =
+                Post.builder()
+                        .id(existingPost.getId())
+                        .user(existingPost.getUser())
+                        .categoryType(existingPost.getCategoryType())
+                        .title(followRequest.title()) // 새 제목 설정
+                        .content(followRequest.content()) // 새 내용 설정
+                        .followType(followRequest.followType())
+                        .createdAt(existingPost.getCreatedAt())
+                        .updatedAt(LocalDateTime.now()) // 업데이트 시각 설정
+                        .postMeta(existingPostMeta)
+                        .build();
+
+        postRepository.save(updatedPost);
```

Comment:
existingPost.update(followRequest.toCommand());

public update(UpdateCommand command) {
    this.categroyType = command.getCategoryType();
///
.///
}